docs/references/api.md

```diff
@@ -25,6 +25,51 @@ Besides importing the Python code directly, consumers who are not writing Python
 }
 ```
 
+## Version Dependencies
+The package name of Ubuntu Pro Client is `ubuntu-advantage-tools`.
+
+The documentation for each endpoint below includes the first version that includes that endpoint.
```

Comment:
```suggestion
The documentation for each endpoint below states the first version to include that endpoint.
```

Reply:
alternatively: "The documentation for each endpoint below includes the first version to introduce that endpoint."